webscrapbook/scrapbook/common/util.py

```diff
@@ -0,0 +1,121 @@
+import os, json, glob, re
+from math import ceil
+from collections import OrderedDict
+from itertools import islice
+
+# string functions
+###############################################################################
+
+def remove_prefix(text, prefix):
+    if text.startswith(prefix):
+        return text[len(prefix):]
+    return text
+
+def remove_suffix(text, suffix):
+    if text.endswith(suffix):
+        return text[:-1 * len(suffix)]
+    return text
+
+def remove_lines(file, count):
+    for _ in range(count):
+        file.readline()
```

Comment:
UnicodeDecodeError here on Windows. Tree files need to be read using UTF-8 encoding.

Reply:
Oops I'll try fix that! I'm not used to windows stuff, so I'll just open all the tree files with `encoding='utf8'`.

Reply:
I think I've fixed it but I'll need you to verify since I don't have access to windows.